src/app/external-cluster-wizard/steps/summary/external-cluster-details/component.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, Input, ViewEncapsulation} from '@angular/core';
+import {ExternalClusterModel} from '@shared/entity/external-cluster';
+
+@Component({
+  selector: 'km-external-cluster-details',
+  templateUrl: './template.html',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
There are no styles in this component so why do we need this encapsulation here? Same comment for all provider specific components below.

Reply:
there are no styles here because all the styles are being handled in their parent component, that's why I've put `ViewEncapsulation.None` here,

Reply:
But `ViewEncapsulation.None` is used to make current component's styles global so if there are no styles here then why do we need it?

Reply:
Ah, I'll change the logic then.